### DISCUSSION TITLE
How do I create a new page?

### DISCUSSION THREAD

**Discussion by SuhaniSuranaArizona** [academicpages/academicpages.github.io#2149] [Q&A] [answered]
I have been trying to create new pages like research etc, for my website but I do not know how do I add content to it? I have made changes to the navigation bar, but want to know on how to make the actual content for that subdirectory?

Thanks!

**Comment by rjzupkoii** [comment]
Do you mean creating a new category, or creating new pages within an existing category?

**Comment by yumzhang17** [comment, marked as answer]
First, you need to modify the navigation.yml, like:
  - title: "Research"
    url: /Resaerch/
Then, in your _pages, you need to have a new html file, like:
research.html
The contents can be something like this:
---
layout: archive
title: "Research"
permalink: /research/
author_profile: true
---

**Reply by erikfred** [reply]
Thanks! This helped me get going with my own customization
